### PR TITLE
[Bridges] add support for Constraint{Primal,Dual}Start to SquareBridge

### PR DESCRIPTION
**Pull request by odow**
Part of https://github.com/jump-dev/MathOptInterface.jl/issues/684

### CODE REVIEW

**Comment by odow** [comment]
@blegat please take a look. You understand better than me.

File: src/Bridges/Constraint/bridges/square.jl [hunk 1]
```diff
@@ -356,3 +413,79 @@ function MOI.get(
...
+            # suppose the constraint is
+            #   [0 x; y 0] in PositiveSemidefiniteConeSquare(2).
+            # If the dual is
+            #   [λ1 λ3; λ2 λ4]
+            # then we have `y λ2 + x λ3` in the Lagrangian.
+            #
+            # In the bridged model, the constraint is
+            #   [0, x, 0] in PositiveSemidefiniteConeTriangle(2).
+            #   [x - y] in Zeros(1)
+            # If the dual is
```
Comment (blegat) [inline src/Bridges/Constraint/bridges/square.jl:455]: I don't think you need to copy-past the comment, just say that we are doing the inverse of what in the getter

File: src/Bridges/Constraint/bridges/square.jl [hunk 2]
```diff
@@ -276,12 +276,37 @@ function MOI.get(::MOI.ModelLike, ::MOI.ConstraintSet, bridge::SquareBridge)
     return bridge.square_set
 end
 
+function MOI.supports(
+    model::MOI.ModelLike,
+    attr::Union{MOI.ConstraintPrimalStart,MOI.ConstraintDualStart},
+    ::Type{SquareBridge{T,F,G,TT,ST}},
+) where {T,F,G,TT,ST}
+    return MOI.supports(model, attr, MOI.ConstraintIndex{F,TT}) &&
```
Comment (blegat) [inline src/Bridges/Constraint/bridges/square.jl:284]: Just return `true`, `AbstractBridgeOptimizer` takes care of taking these into account

Comment (odow) [reply src/Bridges/Constraint/bridges/square.jl]: Actually, I don't think this is true?

**Pull request merged**